src/test/stubs.ts

```diff
@@ -16,6 +16,10 @@ class Range {
 class Location {
     constructor(public uri: URI, public range: Range) {}
 }
+export interface Unsubscribable {
```

Comment:
This is declared but it's not used anywhere AFAICT. Also, you shouldn't need to declare it, you can use rxjs' `Unsubscribable`